lib/apns/core.rb

```diff
@@ -6,20 +6,28 @@ module APNS
   @host = 'gateway.sandbox.push.apple.com'
   @port = 2195
   # openssl pkcs12 -in mycert.p12 -out client-cert.pem -nodes -clcerts
-  @pem = nil # this should be the path of the pem file not the contentes
+  @pems = {} 
+  # this should be the path of the pem file not the contentes
+  # Example:
+  # { some_ios_app : "file/to/pem" }
+
   @pass = nil
 
   class << self
-    attr_accessor :host, :pem, :port, :pass
+    attr_accessor :host, :pems, :port, :pass
+  end
+
+  def self.pem=(default_pem_path)
+    @pems = { default: default_pem_path }
   end
 
-  def self.send_notification(device_token, message)
+  def self.send_notification(device_token, app = :default, message)
     n = APNS::Notification.new(device_token, message)
-    self.send_notifications([n])
+    self.send_notifications(app, [n])
   end
 
-  def self.send_notifications(notifications)
-    sock, ssl = self.open_connection
+  def self.send_notifications(app = :default, notifications)
```

Comment:
not entirely backwards compatible. 
Someone may have had a call lke `APNS.send_notification(notification)` and now it'll interpret the `notification` as if it was the `app`.

If you are adding parameters, it should be added to the end of the signature, or we should have parameters that are `app_or_notifications` and you determine which should be used.